### ISSUE TITLE
`color_print*`-macro: 1) cannot contain trailing commas and 2) using in an expression fails

### ISSUE THREAD

**Issue by trkks** [trkks/terminal_toys#1]
- [x] 1) The following should work but doesn't:
```
color_println!(
  Color::Red,
  "Hello {}",
  "World", // <-- The problem.
);
```

- [x] 2) The following would cause "macro expansion error" or smth on build:
```
match x {
  // ...
  Err(e) => color_println!(Color::Red, "[Server]: Failed to receive data - {}", e),
```
and is forced to be changed into:
```
// ...
  Err(e) => { color_println!(Color::Red, "[Server]: Failed to receive data - {}", e); } ,
```

**Comment by trkks** [comment]
Should be fixed with commit 986dbffedd1ea2f8d5628bbcb73a42bbfcd0f706

**Comment by trkks** [comment]
Problem 1) still remains with the `color_log`-macro.

**Comment by trkks** [comment]
Fixed in 505c44656d2f5a59c1343a36b117a606d3172323 by copying from `color_print` (obviously).
